extensions/frontend/pkg/revaconfig/config.go

```diff
@@ -217,7 +218,7 @@ func FrontendConfigFromStruct(cfg *config.Config) map[string]interface{} {
 							"major":   version.Parsed().Major(),
 							"minor":   version.Parsed().Minor(),
 							"micro":   version.Parsed().Patch(),
-							"string":  version.GetString(),
+							"string":  version.Legacy,
```

Comment:
```suggestion
							"string":  version.LegacyString,
```

https://drone.owncloud.com/owncloud/ocis/11881/39/7
```
And the version data in the response should contain                                    # CapabilitiesContext::checkVersionResponse()

      | name    | value           |

      | string  | %versionstring% |

      | edition | %edition%       |

      | product | %productname%   |
      Failed field string
      Failed asserting that SimpleXMLElement Object &000000003d36bc67000000006f61833c (
          0 => '10.11.0.0'
      ) matches expected '10.11.0'.
```
Maybe that will be fixed by this change?